website/content/en/troubleshooting/_index.md

```diff
@@ -31,16 +31,16 @@ You can use `grep ^E` and `--since` flag from `kubectl` to isolate any errors oc
 To troubleshoot issues with the provider, you can look at logs from the provider pod running on the same node as your application pod
 
 ```bash
-# find the csi-secrets-store-provider-azure pod running on the same node as your application pod
-kubectl get pods -l app=csi-secrets-store-provider-azure -o wide
+# find the secrets-store-provider-azure pod running on the same node as your application pod
+kubectl get pods -l app=secrets-store-provider-azure -o wide -A

```

Comment:
The manifests in this repo use `csi-secrets-store-provider-azure` as the label:

https://github.com/Azure/secrets-store-csi-driver-provider-azure/blob/e11dd11955657b4a731a0ecfeb3ad6bc2127f723/deployment/provider-azure-installer.yaml#L21

Reply:
Seems that there might be inconsistency with the "Managed Add-ons" implementation
This may be intentional; I'll close the PR if so.

`$ kubectl get pods -l app=secrets-store-provider-azure -o wide -A
NAMESPACE     NAME                                     READY   STATUS    RESTARTS   AGE   IP           NODE                                NOMINATED NODE   READINESS GATES
kube-system   aks-secrets-store-provider-azure-hnwd7   1/1     Running   0          27m   10.224.0.6   aks-agentpool-31285785-vmss000000   <none>           <none>
`

`$ kubectl get pod/aks-secrets-store-provider-azure-hnwd7 -n kube-system -o json | jq -r '.metadata. Labels'
{
  "app": "secrets-store-provider-azure",
  "controller-revision-hash": "9475866b6",
  "kubernetes.azure.com/managedby": "aks",
  "pod-template-generation": "1"
}
`

`
$ kubectl get pods -l app=secrets-store-csi-driver -o wide -A
NAMESPACE     NAME                                 READY   STATUS    RESTARTS   AGE   IP           NODE                                NOMINATED NODE   READINESS GATES
kube-system   aks-secrets-store-csi-driver-m8whk   3/3     Running   0          27m   10.244.1.2   aks-agentpool-31285785-vmss000000   <none>           <none>
`

`$ kubectl get pod/aks-secrets-store-csi-driver-m8whk -n kube-system -o json | jq -r '.metadata. Labels'
{
  "app": "secrets-store-csi-driver",
  "controller-revision-hash": "55b67744dc",
  "kubernetes.azure.com/managedby": "aks",
  "pod-template-generation": "1"
}
`

Reply:
```suggestion
kubectl get pods -l '(app in csi-secrets-store-provider-azure, secrets-store-provider-azure)' -o wide -A
```

this should work for the add-on and OSS deployment

Reply:
Minor tweak to that command, validated it against test cluster and updated the pull request.